src/FSharp.Control.Incremental/Transaction.fs

```diff
@@ -7,34 +7,34 @@ open System.Collections.Generic
 [<AutoOpen>]
 module internal LockingExtensions =
     type IAdaptiveObject with
-        /// acquires a write-lock to an AdaptiveObject
+        /// Acquires a write-lock to an AdaptiveObject
         member inline o.EnterWrite() =
             Monitor.Enter o
             while o.ReaderCount > 0 do
                 Monitor.Wait o |> ignore
             
-        /// releases the write-lock to the AdaptiveObject
+        /// Releases the write-lock to the AdaptiveObject
         member inline o.ExitWrite() =
             Monitor.Exit o
         
-        /// determines whether the object is locked and out-of-date
+        /// Determines whether the object is locked and out-of-date
         member inline o.IsOutdatedCaller() =
             Monitor.IsEntered o && o.OutOfDate
 
-/// when evaluating AdaptiveObjects inside a Transaction 
+/// When evaluating AdaptiveObjects inside a Transaction 
 /// (aka eager evaluation) their level might be inconsistent when
 /// attempting to evaluate. Therefore the evaluation may raise
 /// this exception causing the evaluation to be delayed to a later
 /// time in the Transaction.
 exception LevelChangedException of IAdaptiveObject * int * int
```

Comment:
The `IAdaptiveObject` is not consumed at the place where this is caught, can we remove it?

Reply:
I think there's some other code needing it in AdaptiveObject but I'm not entirely sure...
I'll try not to forget it....